ansible/vagrant/Vagrantfile.Solaris10

```diff
@@ -3,6 +3,8 @@
 
 $script = <<SCRIPT
 sudo /opt/csw/bin/pkgutil -y -i python
```

Comment:
do we still need this install too?

Reply:
I doubt it, but Im still testing the nature of the fixes..  I'll test the removal of 2.6 too.